README.md

```diff
@@ -241,7 +258,10 @@ The FOSSASIA Labyrinth allows you to contribute parts to a huge labyrinth. Pleas
 - Comment on an issue that you want to do it. If you have solved several tasks on this game before, you can not claim tasks that are too easy for you because we need them to give others an easy start.
 - Get assigned to the issue you work on, so other people coordinate with you. Being assigned an issue does not mean you can block progress by not answering.
 
+## UI identity guideline
+[>>Click here to read the full UI guideline<<](https://github.com/fossasia/labyrinth/UI_Identity.md)
```

Comment:
This can be a relative link, too.
`[...](UI_Identity.md)`

Reply:
If you add your motivation why you created this, the message is stronger and I think more people will see that it is a good thing to follow.

Reply:
Don't use `>>text<<`, it looks kind of cheap in my opinion.